acvm/src/pwg/hash.rs

```diff
@@ -72,3 +73,42 @@ fn generic_hash_256<D: Digest>(
     let result = hasher.finalize().as_slice().try_into().unwrap();
     Ok(result)
 }
+
+pub fn keccak256(
+    initial_witness: &mut BTreeMap<Witness, FieldElement>,
+    gadget_call: &BlackBoxFuncCall,
+) -> Result<(), OpcodeResolutionError> {
```

Comment:
```suggestion
) -> Result<OpcodeResolution, OpcodeResolutionError> {
```

This needs to match the interface for the other bb function solvers

Reply:
Done